dist/index.css

```diff
@@ -1,8 +1,81 @@
-.next-lib-template_container {
-    background-color: black;
-    color: white
+.next-button_button {
```

Comment:
We have `dist/` in the `.gitignore` but somehow this file ended up in the changes 🤔

Reply:
Nice catch, fixed.

This happens when git is already tracking the file, so if you add that file to `.gitignore` after, it will still continue to be tracked in git. The template already had this file committed in git, so that's why.